transport_nantes/mobilito/views.py

```diff
@@ -391,13 +387,18 @@ def prepare_email(
     return email
 
 
-def get_MobilitoUser(self: Union[TemplateView, FormView]) -> Union[MobilitoUser, None]:
-    """Get the MobilitoUser object for the current user"""
-    if self.request.user.is_authenticated:
+def get_MobilitoUser(request):
```

Comment:
Could have left the return type hint `Union[MobilitoUser, None]` :slightly_smiling_face:

Reply:
Sure, I started to refactor that function, then put it back but not quite all the way clearly.